clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -1094,6 +1094,60 @@ public void testManualAssignmentChangeWithAutoCommitDisabled() {
         consumer.close();
     }
 
+    @Test
+    public void testOffsetOfPausedPartitions() {
```

Comment:
Thanks for adding the test. Would be good to have a more focused test in `FetcherTest` as well for the change there.